### DISCUSSION TITLE
Is it safe for the new update?

### DISCUSSION THREAD

**Discussion by doodooman55** [YimMenu/YimMenu#2520] [Q&A] [answered]
I saw that the file is 2 hours older than the reveal trailer for the new gtao dlc. is yimmenu safe or nah?

**Comment by Yimura** [comment, marked as answer]
It doesn't even work currently, and usually it takes around 30 days for us to know if it's safe to use online.

**Reply by doodooman55** [reply]
Thanks for the quick reply! That is very much appreciated.

**Comment by DSTC69** [comment]
> It doesn't even work currently, and usually it takes around 30 days for us to know if it's safe to use online.
will it be available before then?

**Reply by alexnikou** [reply]
I used it for the new missions (solo session ofc) everything is fine. The only problem is that i can't join with a friend, it says something for the character. I guess it will get fixed

**Reply by duckingaround** [reply]
You managed to inject it? my game just crash over and over after injecting it. any methods you did?
